spec/lib/mohawk/waiter_spec.rb

```diff
@@ -30,10 +30,22 @@ def do_it
 
   context 'waits in between' do
     When do
-      @fake_responses = [false,  true]
+      @fake_responses = [false, true]
       fake_time 0, 1
     end
 
     Then { expect { waiter.wait_until { do_it } }.not_to raise_error }
   end
-end
\ No newline at end of file
+
+  context 'accepts context parameter' do
+    Then { expect { waiter.wait_until(1, 'This is the context') { true } }.not_to raise_error }
```

Comment:
This is how the `wait_until` call looks after adding context as an optional parameter. It looks kind of clunky right now because we always have to pass the timeout.

If we don't want to require the timeout every time then we could switch to named parameters so that it would instead look like `waiter.wait_until(context: 'This is the context')`.